aws/table_aws_resource_explorer_index.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/resourceexplorer2"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAWSResourceExplorerIndex(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_resource_explorer_index",
+		Description: "AWS Resource Explorer Index",
+		List: &plugin.ListConfig{
```

Comment:
@LalitTurbot Can you please add them? I agree, they're not majorly useful right now, but in case they are more useful in the future, it will be helpful to already have it.